ports/usd/portfile.cmake

```
@@ -1,65 +1,130 @@
 # Don't file if the bin folder exists. We need exe and custom files.
 set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
 
-message(STATUS [=[
-The usd port does not work with the version of Threading Building Blocks (tbb) currently chosen by vcpkg's baselines,
-and does not expect to be updated to work with current versions soon. See
-https://github.com/PixarAnimationStudios/USD/issues/1600
-
-If you must use this port in your project, pin a version of tbb of 2020_U3 or older via a manifest file.
-See https://vcpkg.io/en/docs/examples/versioning.getting-started.html for instructions.
-]=])
-
 string(REGEX REPLACE "^([0-9]+)[.]([0-9])\$" "\\1.0\\2" USD_VERSION "${VERSION}")
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO PixarAnimationStudios/OpenUSD
-    REF "v${USD_VERSION}"
-    SHA512 e510f6421caba5e74c6efe5b56b17e9c9741ece0cfd5020148ca89b3ac32bd8781ab00dfc7a134163c85af3f4f01f2529a9baa5a9df9b0c80cbca003e6d199e2
-    HEAD_REF master
+    REF "v${VERSION}"
+    SHA512 7d4404980579c4de3c155386184ca9d2eb96756ef6e090611bae7b4c21ad942c649f73a39b74ad84d0151ce6b9236c4b6c0c555e8e36fdd86304079e1c2e5cbe
+    HEAD_REF dev
```

```suggestion
    HEAD_REF release
```
The default branch of openusd is named release.

---

Thanks for the review, can you clarify the purpose of `HEAD_REF` is it to points to the latest bleeding edge dev commit (here `dev` for OpenUSD), or the latest stable repository commit (here `release` for OpenUSD)

---

https://learn.microsoft.com/en-us/vcpkg/maintainers/functions/vcpkg_from_github#head_ref